DEVELOPER_GUIDE.md

```diff
@@ -59,7 +59,7 @@ Fork ml-commons by clicking the "Fork" button at the top of the [GitHub reposito
 Clone your forked version of ml-commons to your local machine (replace `opensearch-project` in the command below with your GitHub username):
 
 ```bash
-$ git clone git@github.com:opensearch-project/opensearch-py-ml.git
+$ git clone git@github.com:opensearch-project/ml-commons.git
```

Comment:
This code change is in 2.x. Can you help backport 2.x code change to feature/conversation branch first? Then you can rebase your PR.

Reply:
created a PR for that: https://github.com/opensearch-project/ml-commons/pull/1212

Reply:
Is PR #1212 going to merge the feature code to 2.x?

I mean "Can you help backport 2.x code change to feature/conversation branch first?". This is a big PR, we should develop it on feature/conversation branch first.